parcel-bundler/parcel
# Upgrade 'ws' package to 7.x and add flow typings

This updates all of our direct use of the `ws` package to 7.x and adds flow typings. It also addresses violations introduced by adding these typings.

Test Plan: 
* Run the react-hmr example. 
* Verify changes are reflected live. 
* Cause an error, verify a message is shown live. 
* Fix the error, verify the message is removed live.
